core/src/main/java/nl/nn/adapterframework/doc/GenericOptionAttributeTask.java

```diff
@@ -0,0 +1,17 @@
+package nl.nn.adapterframework.doc;
```

Comment:
Idem, geen license informatie

Reply:
Gedaan. Maar ik heb niet gekeken of ik "2020, 2021" of alleen "2021" moet neerzetten.